issues/2018-03-19/issue_2018-03-19.md

```diff
@@ -5,42 +5,48 @@
 
 ## Open Source Contribution - A Case Report 
 
-The cool thing about open source software is its openness. Not only that everyone is able to understand and reproduce the code, but everyone can participate and contribute - this is the biggest strength of FOSS and also what many projects depend on. You might find yourself in a situation where you have the possibility to contribute sooner than you think.
+The cool thing about open source software is its openness. Not just that everyone can examine the code, and reproduce it, but that everyone can participate and contribute to it’s growth; this is the biggest strength of FOSS and also the principle that many projects depend on.
 
-Diceware is a convenient tool to create strong random passwords. Bhavin found an uncaught exception and decided to commit a fix. To write the code and put a pull request is however just the start. Read [Bhavin's blog post about this pull request](https://geeksocket.in/blog/contributing-diceware-tool/) for all details about the exciting progress from discovering a bug to get the own code merged into the project, making it a little better.
+Who knows, you might find yourself in a situation where you have the possibility to contribute sooner than you think.
 
-It's encouraging to see how core developers take care, make suggestions and guide contribuors to deliver a professional fix. The things you learn and experience during a process like this are priceless.
+Here’s a recent example. Diceware is a convenient tool to create strong random passwords. Bhavin found an uncaught exception,  and decided to commit a fix.
 
-A very much worth reading blog post for everyone who's interested in how open source realley works. 
+To write the code and create a pull request is however just the start. Read [Bhavin's blog post about this pull request](https://geeksocket.in/blog/contributing-diceware-tool/) for all details about the exciting progress from discovering a bug to get the own code merged into the project, making it a little better.
```

Comment:
I feel `to getting the code fix merged` makes more sense. @jasonbraganza views?

Reply:
Another typo I missed. I’d rather have it be ```their own code merged````
Gives the contributor (Bhavin) a better sense of ownership

Reply:
Another typo I missed. I’d rather have it be ```their own code merged```
Gives the contributor (Bhavin) a better sense of ownership

Reply:
Once you push that change, I think it's good to be merged.